README.md

```diff
@@ -37,11 +37,12 @@ python -m unittest discover -s tests
 Start by making a copy of `.env.template` and renaming it `.env`. For the backend to function, you will need to add the missing values:
 
 * **NACHET_AZURE_STORAGE_CONNECTION_STRING**: Connection string to access external storage (Azure Blob Storage).
-* **NACHET_MODEL_ENDPOINT_REST_URL**: Endpoint to communicate with deployed model for inferencing. 
+* **NACHET_MODEL_ENDPOINT_REST_URL**: Endpoint to communicate with deployed model for inferencing.
 * **NACHET_MODEL_ENDPOINT_ACCESS_KEY**: Key used when consuming online endpoint.
 * **NACHET_DATA**: Url to access nachet-data repository
 * **NACHET_HEALTH_MESSAGE**: Health check message for the server.
+* **NACHET_MAX_CONTENT_LENGHT**: Set the maximum size of the file that can be uploaded to the backend. Need to be the same size as the `proxy body size` variable from DevSecOps.
```

Comment:
```suggestion
* **NACHET_MAX_CONTENT_LENGTH**: Set the maximum size of the file that can be uploaded to the backend. Needs to be the same size as the `client_max_body_size` [value](https://github.com/ai-cfia/howard/blob/dedee069f051ba743122084fcb5d5c97c2499359/kubernetes/aks/apps/nachet/base/nachet-ingress.yaml#L13) set from the deployment in Howard.
```

Reply:
The hyperlink doesn't seem to work in suggested changes but I updated the variable since it wasn't the right one.